src/components/Table.tsx

```diff
@@ -119,6 +119,9 @@ function Table () {
         {
           props => (
             <div className="p-4 overflow-x-auto">
+              <h1>
+                <Trans>Welcome</Trans>
+              </h1>
```

Comment:
Do we actually need this heading here?

I will merge the PR with my adjustments, we can discuss this later.

Reply:
I would argue the same. Instead, we could save the space for the extra headlines and highlight (boldface?) the corresponding active menu point ("Plays", "Locations", "Originals", or "About").

Reply:
I agree. Let's get rid of headings altogether. Whatever will make pages more consistent.